modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -2,29 +2,29 @@
 = Single Sign-On (SSO)
 :description: SSO allows you to log in to the Aura Console using their company IdP credentials.
 
-label:AuraDB-Virtual-Dedicated-Cloud[]
-label:AuraDS-Enterprise[]
-label:AuraDB-Business-Critical[]
+* *AuraDB Virtual Dedicated Cloud and AuraDS Enterprise* Supports both Organization SSO and Instance SSO which are configurable in the Aura console. 
```

Comment:
I believe organization SSO is available to all orgs. Will confirm this.

Reply:
I started questioning this logic, because we don't have a technical reason for this limitation.

In the next year, we plan to remove this restriction so that org SSO is available for all orgs.

Reply:
Good stuff, SSO for all orgs will be a better user experience!